specification/configuration.md

```diff
@@ -89,6 +90,52 @@ environment variables:
   If `SPLUNK_CONFIG` is defined then these options are not required but MAY be
   defined.
 
+### Kubernetes
+
+While Kubernetes supports container technology that can be configured using
+environment variables, package management solutions such as Helm charts and
+Operators require YAML-based configuration. As a result, Kubernetes package
+management solutions MUST support the following YAML configuration options:
+
+- `clusterName` ()             : [REQUIRED] Name of the cluster.
```

Comment:
This is actually required only if Observability destination is enabled. I think we should mention that

Reply:
Does it make sense to the authors of Kuberntetes package management solutions what required means? Do we need to define behavior for them when a required variable is not provided?

Reply:
Addressed your comment @MrAlias . @dmitryax I think we should be consistent -- either both require or both don't.

Reply:
Ok, I'll check on this with SCK team